msdk-featdet/msdk-featdet-openms/src/main/java/io/github/msdk/featdet/openms/FeatureFinderMetaboMZValueDetector.java

```diff
@@ -0,0 +1,83 @@
+package io.github.msdk.featdet.openms;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.annotation.Nonnull;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.NodeList;
+import org.w3c.dom.Node;
+import org.w3c.dom.Element;
+import org.xml.sax.SAXException;
+
+import io.github.msdk.datamodel.rawdata.RawDataFile;
+
+public class FeatureFinderMetaboMZValueDetector {
```

Comment:
I'd call the class simply FeatureFinderMetaboDetector, as "MZValue" can be a bit confusing (I can explain why if you're interested)